### ISSUE TITLE
Issue when renderTexture.erase entries is a list

### ISSUE THREAD

**Issue by jbgomez21** [photonstorm/phaser#4528]
## Version

* Phaser Version: 3.17.0

* Operating system: all

* Browser:

## Description

Before 3.17.0 
```
draw: function(...) {
this.batchList(entries, x, y, alpha, tint);
}
```

After 3.17.0
```
draw: function(...) {
this.batchList(entries, x + this.frame.cutX, y + this.frame.cutY, alpha, tint);
} 
```

When I do: 
```
const erasers = [....];
this.renderTexture.erase(erasers);
```

```
  batchGameObjectWebGL: function (gameObject, x, y)
    {
        if (x === undefined) { x = gameObject.x; }
        if (y === undefined) { y = gameObject.y; }

        var prevX = gameObject.x;
        var prevY = gameObject.y;

        if (!this._eraseMode)
        {
            this.renderer.setBlendMode(gameObject.blendMode);
        }

        gameObject.setPosition(x, y);

        gameObject.renderWebGL(this.renderer, gameObject, 0, this.camera, null);

        gameObject.setPosition(prevX, prevY);
    },
```

Before 3.17.0 batchGameObjectWebGL is called from draw with x and y with undefined values, therefore it's initialized with gameObject.x and gameObject.y and all is OK!

After 3.17.0 batchGameObjectWebGL is called from draw with x and y with NaN values then it is not initialized with gameObject.x and gameObject.y therefore does not erase

**Comment by telinc1** [comment]
This also affects the `draw` method and applies even if the entries are not a list. The actual bug is that the `x` and `y` parameters are no longer optional because of the math done when `batchList` is called. This breaks examples such as [Graphics to Render Texture](http://labs.phaser.io/view.html?src=src/game%20objects/render%20texture/graphics%20to%20render%20texture.js) and [Explosion](http://labs.phaser.io/view.html?src=src/game%20objects/render%20texture/explosion.js). Both of them work on 3.16.2, but not on 3.17.0. Editing them to specify explicit positions (for example, by changing line 33 of Graphics to Render Texture to `rt.draw(graphics, graphics.x, graphics.y);`) will fix them.

**Comment by photonstorm** [comment]
Just had a look at this, and you're right, it broke when I merged in #4074 which didn't account for this. Hmmm.. all to do with the frame stuff that was added. Will see if there's an easy solution, otherwise may unpick that.

**Comment by photonstorm** [comment]
Thank you for submitting this issue. We have fixed this and the fix has been pushed to the `master` branch. It will be part of the next release. If you get time to build and test it for yourself we would appreciate that.
